package.json

```diff
@@ -12,8 +12,8 @@
 		"eslint": "^7"
```

Comment:
I guess we need to update the peer dependency too if the goal is to support eslint 8

Reply:
If the plug-in works with both we can say 7 || 8 though. What do you think?

Reply:
In the `fintech-seller-dashboard` repo, we've got this PR: https://github.com/Tradeshift/fintech-seller-dashboard/pull/926

While on eslint version 7.32.0, the latest versions of the eslint plugin and parse are not compatible.

Good point, I'll update the peer dependency to eslint 8

Reply:
Ah, I just confirmed `@typescript-eslint/parser@5.12.0` is compatible with `eslint@7.9.0`, so I'll add `7 || 8`